com.unity.cinemachine/Tests/Runtime/CamerasBlendingTests.cs

```diff
@@ -1,3 +1,4 @@
+#if !CINEMACHINE_HDRP
```

Comment:
I would suggest using this way instead 
https://github.com/Unity-Technologies/com.unity.cinemachine/blob/release/2.9/Projects/HDRP/Assets/Editor/InitializeTests.cs

https://docs.unity3d.com/Packages/com.unity.test-framework@1.1/manual/reference-attribute-conditionalignore.html

Will still run the test but will be ignored in the pipeline. It will make it easier for us to remember there is a problem here.